base/src/main/java/io/spine/validate/constraint/ConstraintOf.java

```diff
@@ -28,13 +28,13 @@
 import java.util.Optional;
 
 /**
- * An external validation for a field.
+ * An external constraint for a field.
  *
- * <p>Contains information about validation of another field, described by the option value.
+ * <p>Contains information about constraint of another field, described by the option value.
  */
-final class ValidationOf extends MessageOption<String> {
+final class ConstraintOf extends MessageOption<String> {
 
-    ValidationOf() {
+    ConstraintOf() {
         super(OptionsProto.validationOf);
```

Comment:
We will need to rename the Protobuf option as well. To me, `constraints_for` looks good:

```proto
message UkraineOnly {
    option (constraint_for) = "com.acme.User.country";
    
    string name = 1 [(pattern).regex = "^Ukraine$"];
}
```
@alexander-yevsyukov @dmdashenkov FYI.